src/cosalib/cmdlib.py

```diff
@@ -191,7 +191,7 @@ def import_ostree_commit(repo, commit, tarfile, force=False):
             and not force):
         return
 
-    with tempfile.TemporaryDirectory(dir=f'{repo}/tmp') as d:
+    with tempfile.TemporaryDirectory(dir=f'{repo}') as d:
```

Comment:
From your commit message:
```
Note that `dir=f{repo}` just means to create the temporary directory in
which we extract there; we're not directly extracting on top of existing
OSTree repo.
```

That might be a good comment to have in the code here because "directly extracting on top of the existing OSTree repo" is exactly the behavior I thought you intended when I read the code. A comment clearing it up would be useful.

Reply:
minor `dir=f'{repo}'` -> `dir=repo` - no need for an f string since we're not adding any text to the value of the variable?

Reply:
Done! :arrow_down: